src/server/server.babel.js

```diff
@@ -1,21 +1,3 @@
-const ignorePatterns = [
```

Comment:
This has been driving me nuts. Let's create an issue. I suspect it's because the SC worker process isn't exiting.

I want to track this down either via a `git bisect` and a test script or adding a hook on exit that looks at node's run queue.